src/web/app/pages-admin/admin-search-page/admin-search-page.component.spec.ts

```diff
@@ -32,6 +32,7 @@ const DEFAULT_STUDENT_SEARCH_RESULT: StudentAccountSearchResult = {
   googleId: 'googleId',
```

Comment:
It would be nice if we could add a snapshot test for when `isCourseDeleted` is true as well (or modify one of the existing ones).